.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,40 @@
+| Q | A
+| --- | ---
+| Bug fix? | no
+| New feature? | no
+| BC breaks? | no
+| Deprecations? | no
+| Fixed tickets | fixes #issuenum
+| Related issues/PRs | #issuenum
+| License | MIT
+| Documentation PR | sulu/sulu-docs#prnum
+
+#### What's in this PR?
+
+Explain the contents of the PR.
+
+#### Why?
+
+Which problem does the PR fix? (remove this section if you linked an issue above)
+
+#### Example Usage
+
+~~~php
+// If you added new features, show examples of how to use them here
+// (remove this section if not a new feature)
+
+$foo = new Foo();
+
+// Now we can do
+$foo->doSomething();
+~~~
+
+#### BC Breaks/Deprecations
+
+Describe BC breaks/deprecations here. (remove this section if not needed)
+
+#### To Do
+
+- [ ] Create sulu/sulu PR
```

Comment:
Maybe something like `Move changes to sulu/sulu develop branch`, where one part of that is a link to the specific folder in github? Just to be very clear what is meant here.